lms/djangoapps/commerce/models.py

```diff
@@ -1,3 +1,19 @@
-"""
-This file is intentionally empty. Django 1.6 and below require a models.py file for all apps.
-"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.models import ConfigurationModel
+
+
+class CommerceConfiguration(ConfigurationModel):
+    """ Commerce configuration """
+
```

Comment:
I may have initially wrote this code as a prototype; but, if you are submitting the PR @vkaracic, it's your code.

That being said, yes, this would be the place to add multi-tenancy support. That should be tested extensively (obviously).